### ISSUE TITLE
Refactor code and create utils/aosp_build_tag.sh

### ISSUE THREAD

**Issue by jcfr** [kylemanna/docker-aosp#12]
The idea is to avoid code duplication in `tests/build-kitkat.sh` and `tests/build-lollipop.sh`

**Comment by kylemanna** [comment]
True.  I assume that people really using this will fork the build scripts to something more suitable for their own builds.  I keep them around as a demonstration for those investigating the Docker image.

That said, de-duping the code is always nice.
